.travis.yml

```diff
@@ -18,7 +18,7 @@ install:
 #
 script:
     - diff -u <(echo -n) <(gofmt -d `find . -name '*.go' | grep -v /vendor/`)
-    - go list ./... | grep -v /vendor/ | xargs -L1 golint
+    - go list ./... | grep -Ev '/vendor/|/migration' | xargs -L1 golint
```

Comment:
`/migration` contains the generated file from `go-bindata` which doesn't adhere to linting rules